README.md

```diff
@@ -124,43 +156,48 @@ Crash reports sent to Raygun from this Serilog sink will include HTTP context de
 
 Setting `ignoredFormFieldNames` to a list that only contains "*" will cause no form fields to be sent to Raygun. Placing * before, after or at both ends of an entry will perform an ends-with, starts-with or contains operation respectively.
 
-Note that HTTP headers, query parameters, cookies, server variables and raw request data can also be filtered out. Configuration to do so is described in the [RaygunSettings](#raygun4net-features-configured-via-raygunsettings) section further below.
+Note that HTTP headers, query parameters, cookies, server variables and raw request data can also be filtered out. Configuration to do so is described in the [RaygunSettings](https://raygun.com/documentation/language-guides/serilog-sinks/crashreporting/installation/#raygun4net-features-configured-via-raygunsettings) section further below.
```

Comment:
Do we want this to link to the raygun docs or just stay within github?

Reply:
Removed